chains/evm/contracts/onRamp/OnRamp.sol

```diff
@@ -463,9 +483,9 @@ contract OnRamp is IEVM2AnyOnRampClient, ITypeAndVersion, Ownable2StepMsgSender
     if (extraArgs.length >= 4 && bytes4(extraArgs[:4]) == ExtraArgsCodec.GENERIC_EXTRA_ARGS_V3_TAG) {
       resolvedArgs = ExtraArgsCodec._decodeGenericExtraArgsV3(extraArgs);
 
-      if (resolvedArgs.tokenReceiver.length != 0) {
-        this.validateDestChainAddress(resolvedArgs.tokenReceiver, destChainConfig.addressBytesLength);
-      }
+      resolvedArgs.tokenReceiver = resolvedArgs.tokenReceiver.length != 0
+        ? _validateDestChainAddress(resolvedArgs.tokenReceiver, destChainConfig.addressBytesLength)
+        : resolvedArgs.tokenReceiver;
```

Comment:
Could do:
```suggestion
if (resolvedArgs.tokenReceiver.length != 0) {
  resolvedArgs.tokenReceiver =  _validateDestChainAddress(resolvedArgs.tokenReceiver, destChainConfig.addressBytesLength)
}
```
Since the `else` of the ternary doesn't do anything

Reply:
[efe68dc](https://github.com/smartcontractkit/chainlink-ccip/pull/1476/commits/efe68dca2d4bf48f572b73cc81a745b1a4f33b06)